src/components/cards/PreCombatCardTemplate.tsx

```diff
@@ -1,61 +0,0 @@
-import React from "react";
-import { enemyColor, friendlyColor } from "../../gameData";
-import { ArmyColors, BaseUnit, Unit, UnitCounts, UnitType } from "../../types";
-
-interface PreCombatCardTemplateProps {
-  armyStyle: "friendly" | "enemy";
-  headerText: "Your Army" | "Enemy Army";
-  BASE_UNIT_DATA: BaseUnit;
```

Comment:
If this is only being used to determine the background image, then prefer not to pass in the entire `BaseUnit` but instead something like `bgImage: BaseUnit["bgImageSm"]`. More precise. Also YELLING IS NOT FUN